graylog2-web-interface/src/components/configurationforms/NumberField.jsx

```diff
@@ -69,20 +69,20 @@ const NumberField = createReactClass({
   },
 
   render() {
-    const { typeName } = this.props;
-    const { field } = this.props;
+    const { field, title, typeName } = this.state;
```

Comment:
It seems the issue is caused by this line here it should be `this.props` instead of `this.state`.

Reply:
Fixed it, that was a mistake on my side.